amaltheia/update.py

```diff
@@ -95,10 +95,14 @@ class AptPackagesUpdater(Updater):
     }"""
 
     def update(self):
+        with_new_pkgs_flag = jinja(self.updater_args.get('withnewpkgs', False))
```

Comment:
I would suggest changing the option name to `with-new-pkgs` to stay consistent with the rest of the argument names

Reply:
done